src/task.c

```diff
@@ -754,11 +754,12 @@ DLLEXPORT void gdblookup(ptrint_t ip)
     frame_info_from_ip(&func_name, &line_num, &file_name, ip, 0);
     if (func_name != NULL) {
         if (line_num == ip)
-            jl_safe_printf("unknown function (ip: %d)\n", line_num);
+            jl_safe_printf("unknown function (ip: %p)\n", (void*)ip);
```

Comment:
If it *is* a pointer, and only holds pointers, why is it declared as an integer in the first place, and not as a void *?  That's what doesn't seem to make sense to me...